lib/presentation/util/helper/responsive_widget.dart

```diff
@@ -36,8 +36,10 @@ class ResponsiveWidget extends StatelessWidget {
   final Widget mediumScreen;
   final Widget smallScreen;
 
-  static final int minLargeWidth = 960;
-  static final int minMediumWidth = 640;
+  static final int minLargeWidth = 950;
+  static final int minMediumWidth = 600;
+
+  static final double horizontalPaddingLargeWidth = 132.0;
```

Comment:
can you explain to me?

Reply:
For a large tablet screen by design, we need to indent the listview items in the middle. So I set the value to all widgets that I need to use.